packages/grafana-data/src/transformations/transformers.ts

```diff
@@ -34,7 +34,9 @@ export const standardTransformers = {
   reduceTransformer,
   concatenateTransformer,
   calculateFieldTransformer,
-  seriesToColumnsTransformer,
+  joinByFieldTransformer,
+  /** @deprecated */
+  seriesToColumnsTransformer: joinByFieldTransformer,
```

Comment:
If I am understanding this correctly we are removing the old `seriesToColumnsTransformer` implementation and stubbing it with `joinByFieldTransformer` implementation. To allow for backwards compatibility we are deprecating the original option to eventually remove it in a future version in favor of keeping just `joinByFieldTransformer` correct?

Reply:
yup -- the name `seriesToColumns` is so old and does not really make sense anymore.  This has been called "Join (outer)" or "timeseries join" for so long that it is confusing to find it by the internal name "seriesToColumns" (a name that make sense in grafana 6)